test/fuzzer.cpp

```diff
@@ -39,8 +41,10 @@ string contains(string const& _haystack, vector<string> const& _needles)
 	return "";
 }
 
-int main()
+int main(int argc, char *argv[])
 {
+	bool standard = ((argc > 1) && (string(argv[1]) == "--standard-json")) ? true : false;
```

Comment:
Can you use the tools I introduced here https://github.com/ethereum/solidity/pull/2185/files ?
(Now it's me who has the unpublished pull request ;-) )